packages/components/src/VirtualizedList/ListTable/ListTable.component.js

```diff
@@ -54,7 +54,7 @@ function ListTable(props) {
 	return (
 		<VirtualizedTable
 			className={`tc-list-table ${theme['tc-list-table']}`}
-			gridClassName={theme.grid}
+			gridClassName={`${theme.grid} tc-dropdown-container`}
```

Comment:
use a constant pour target this class tc-dropdown-container and reuse it in https://github.com/Talend/ui/pull/1403/files#diff-f131276a4f6398606943e59f5ea4a345R127